integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXEncryptedVolumeTest.java

```diff
@@ -222,7 +223,7 @@ public void testCreateDistroXWithEncryptedVolumesInSingleRG(TestContext testCont
 
         testContext
                 .given(EnvironmentNetworkTestDto.class)
-                    .withServiceEndpoints()
+                    .withServiceEndpoints(ServiceEndpointCreation.DISABLED)
```

Comment:
Why was this required?

Reply:
Because of at [AZURE provider](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/azure/AzureCloudProvider.java#L226-L234):
```
@Override
public ServiceEndpointCreation serviceEndpoint() {
    ServiceEndpointCreation serviceEndpointCreation = ResourceGroupTest.isSingleResourceGroup(getTestParameter().get(ResourceGroupTest.AZURE_RESOURCE_GROUP_USAGE))
                        ? ServiceEndpointCreation.ENABLED_PRIVATE_ENDPOINT
                        : ServiceEndpointCreation.DISABLED;
    LOGGER.debug("Azure service endpoint creation: {}", serviceEndpointCreation);
    return serviceEndpointCreation;
}
```
for `SINGLE` resource group usage the `ServiceEndpointCreation` is going to be `ENABLED_PRIVATE_ENDPOINT`. That is causing issue in case of a general AZURE E2E test: `TestFailException: Cluster 'azure-test-c8fdbb933939455a9' entered into creation failed state. Status: '{status=CREATE_FAILED}' statusReason: '{statusReason=The provided private DNS zone id  for service Microsoft.DBforPostgreSQL/servers is not a valid azure resource id.}'`